### ISSUE TITLE
adobe reader cannot scroll

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-frontend#49]
Testing was done using the JAWS 2019.1810.64 - October 2018 release of Jaws Screen Reading Software.

•	Cannot scroll through Adobe reader preview for items

**Comment by kimpham54** [comment]
this ticket needs additional clarification

**Comment by jrynhart** [comment]
We do not use Adobe reader - unsure if this is referring to UniversalViewer
Noticed in UniversalViewer, in thumbnail preview panel, that the individual thumbnails can not be tabbed
If that is a required upgrade, will open another ticket for it
